src/main/java/org/jabref/gui/preferences/keybindings/presets/NewEntryBindingPreset.java

```diff
@@ -0,0 +1,40 @@
+package org.jabref.gui.preferences.keybindings.presets;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jabref.gui.keyboard.KeyBinding;
+
+public class NewEntryBindingPreset implements KeyBindingPreset {
+
+    private static final Map<KeyBinding, String> KEY_BINDINGS = new HashMap<>();
```

Comment:
Since it is only used in one method, I would propose to generate the map in getKeyBindings directly (static variables always allocate memory and this one here is used only rarely).

Reply:
It's `Map.of`, isn't it?

Reply:
Using `put` is fine (but map.of might also work). What I meant is that there is no need to store these keybindings in a static final variable; just put them directly in the `getKeyBindings` method.